src/fuzzer.rs

```diff
@@ -266,7 +266,7 @@ pub fn fuzz() -> Result<()> {
         &mut fuzzer,
         &mut state,
         &mut mgr,
-        Duration::from_millis(config.request_timeout), // TODO: implement a proper timeout, this is not a request timeout but an input (sequence timeout)
+        Duration::from_millis(0), // disable the timeout
```

Comment:
Does that work? Not 100% sure

Reply:
At first I doubted this as well, but it seems to work. Is that intended behaviour or should we introduce a change in LibAFL to support this explicitly?

Reply:
the doc says nothing about it
but with glibc,
https://github.com/lattera/glibc/blob/master/sysdeps/mach/hurd/setitimer.c#L187C1-L187C63
yes this works

Reply:
It might not work on some systems thoguh, right? Especially if it's not documented..
Also, what about windows etc?
We can try to introduce this to LibAFL but then we probably don't want to pass 0 but have a `no_timeout` fn

Reply:
oh wait, sorry. setitimer() is just used for macos.
for any Linux systems, we use timer_create()/timer_settime() API, and [here](https://man7.org/linux/man-pages/man2/timer_settime.2.html)it says,

```
 If new_value->it_value specifies a zero value
       (i.e., both subfields are zero), then the timer is disarmed.
```

so, in summary, for linux this should work, and for non-linux (where we use setitimer()) it's not documented